test/framework/src/main/java/org/elasticsearch/index/mapper/MapperScriptTestCase.java

```diff
@@ -26,23 +27,29 @@
 
     protected abstract FactoryType errorThrowingScript();
 
-    protected FactoryType compileScript(String name) {
-        throw new UnsupportedOperationException("Unknown script " + name);
-    }
+    protected abstract FactoryType singleValueScript();
+
+    protected abstract FactoryType multipleValuesScript();
 
     @Override
     @SuppressWarnings("unchecked")
-    protected final <T> T compileScript(Script script, ScriptContext<T> context) {
+    protected <T> T compileScript(Script script, ScriptContext<T> context) {
         if (script.getIdOrCode().equals("serializer_test")) {
-            return (T) serializableScript();
+            return (T)serializableScript();
```

Comment:
nit: I definitely prefer a space here...

Reply:
this was not meant to happen :)